### DISCUSSION TITLE
Splitting up MD testing for faster turnaround

### DISCUSSION THREAD

**Discussion by alexhroom** [ISISNeutronMuon/MDMC#677] [Ideas]
For a faster turnaround on pull request tests, we are considering splitting the MD testing into 'fast' tests and more complex tests. The fast tests will run on a PR, with more complex ones as a cron job. Does anyone with more experience into the MD tests have an idea of how well this split may work?

**Comment by Anders-Markvardsen** [comment]
This is a super suggestion and thanks for implementing it:)

**Comment by franzlang** [comment]
this has now been implemented as part of the following issues and PRs:
Travis: https://github.com/MDMCproject/MDMCv0.2_pilot/issues/671 and https://github.com/MDMCproject/MDMCv0.2_pilot/pull/712
GitHub Actions: https://github.com/MDMCproject/MDMCv0.2_pilot/issues/749 and https://github.com/MDMCproject/MDMCv0.2_pilot/pull/752
